MathLib/mechanics_invariants.h

```diff
@@ -32,89 +32,87 @@ struct Invariants final
 
     /// Determinant of a matrix in Kelvin vector representation.
     static double determinant(
-        Eigen::Matrix<double, KelvinVectorSize, 1> const& vec);
+        Eigen::Matrix<double, KelvinVectorSize, 1> const& v);
 
     /// Inverse of a matrix in Kelvin vector representation.
     static Eigen::Matrix<double, KelvinVectorSize, 1> inverse(
-        Eigen::Matrix<double, KelvinVectorSize, 1> const& vec);
+        Eigen::Matrix<double, KelvinVectorSize, 1> const& v);
 
     static double effectiveStress(
-        Eigen::Matrix<double, KelvinVectorSize, 1> const& deviatoric_stress)
+        Eigen::Matrix<double, KelvinVectorSize, 1> const& v)
     {
-        return std::sqrt(3 * J2(deviatoric_stress));
+        return std::sqrt(3 * J2(v));
     }
 
-    static double J2(
-        Eigen::Matrix<double, KelvinVectorSize, 1> const& deviatoric_stress)
+    static double J2(Eigen::Matrix<double, KelvinVectorSize, 1> const& v)
```

Comment:
Here and below it would be nice if the argument name reflected that a deviatoric tensor is expected.